src/OMSimulatorLib/Model.cpp

```diff
@@ -208,9 +208,6 @@ oms_status_enu_t oms::Model::importSnapshot(const char* snapshot_, char** newCre
   ComRef new_cref = ComRef(ssdNode.attribute("name").as_string());
   std::string ssdVersion = ssdNode.attribute("version").as_string();
 
-  if (new_cref != getCref())
```

Comment:
Check here if `new_cref` is already in scope or not.

Reply:
@this is fixed with https://github.com/OpenModelica/OMSimulator/pull/1006/commits/174b6d09a23fd39d49d4b5ea57ad9dc64fdbc982